Documentation/Books/Manual/DataModeling/OperationalFactors.md

```diff
@@ -0,0 +1,141 @@
+
+Number of Databases and Collections
+-----------------------------------
+
+Sometimes you can consider to split up data over multiple collections. For example one could create a
+new set of collections for each new customer, instead of having a customer field on each documents. Having a
+large number of collections has no performance penalty and results in good performance.
+
+Grouping documents into collections by type (i.e. a session collection 'sessions_dev', 'sessions_prod') allows
+you to avoid an index on the type field.
+
+A few things to consider:
+- Adding an extra collection always incurs a small amount of overhead for the collection metadata and indexes.
+- Only with the _MMFiles storage engine_, creating extra databases will require two compaction and cleanup threads per database.
+  This might lead to undesirable effects should you decide to create many databases compared to the number of available CPU cores.
+
+Cluster Sharding
+----------------
+
+The ArangoDB cluster _partitions_ your collections into one or more _shards_ across multiple _DBServers_.
+This enables efficient _horizontal scaling_: It allows you to store much more data, since ArangoDB distributes the data
+automatically to the different servers. In many situations one can also reap a benefit in data throughput,
+again because the load can be distributed to multiple machines.
+
+ArangoDB uses the specified _shard keys_ to determine in which shard a given document is stored. Choosing the right
+shard key can have significant inpact on your performance can reduce network traffice and increase performance,
+
+ArangoDB uses consistent hashing to compute the target shard from the given values (as specified via 'shardKeys').
+The ideal set of shard keys allows ArangoDB to distribute documents evenly across your shards and your _DBServers_.
+By default ArangoDB uses the `_key` field as a shard key. For a custom shard key
+you should consider a few different properties:
+
+- **Cardinality**: The cardinality of a set is the number of distinct values that it contains. A shard key with
+  only _N_ distinct values can not be hashed onto more than _N_ shards. Consider using multiple shard keys, if one of your values has a low cardinality.
+- **Frequency**: Consider how often a given shard key value may appear in your data. Having a lot of documents with identical shard   keys will lead to unevenly distributed data.
+
+See [Sharding](../Architecture/DeploymentModes/Cluster/Architecture.md#Sharding) for more information
+
+Document and Transaction Size
+-----------------------------
+
+When designing your data-model you should keep in mind that the size of documents affects the performance
+in the single-server and cluster setup.
+Each document has a certain amount of overhead, depending on the storage engine and the indexes you added
+to the collection. The overhead may become significant if your store a large amount of very small documents.
```

Comment:
Should rather start with something like "huge quantities of small documents not good, but huge document are bad too". Then go into the details for each and how it should ideally be.